File: py/1732.py
class Solution:
    def largestAltitude(self, gain: List[int]) -> int:
        res = [0]
        max1 = 0
        for i in range(1,len(gain)+1):
            if sum(gain[:i])>max1:
                max1 = sum(gain[:i])
            
        return max1

File: cpp/1732.cpp
class Solution {
public:
    int largestAltitude(vector<int>& gain) {
        vector<int> res = {0};
        int max2 = 0;
        int max1 = 0;
        for (int i=1;i<gain.size()+1;i++){
            max2 = accumulate(gain.begin(),gain.begin()+i,0);
            if (max2>max1){
             max1 = max2;   
            } 
            }
        return max1;
    }
};
